content/_help/get-started/the-deadline-to-verify-my-identity-in-person-passed._en.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: help
+title: The deadline to verify my identity in person passed
+category: get-started
```

Comment:
Should this be in the "Verify your identity" category?

Reply:
Yes, for now, this should be in the "verify your identity" category. There is separate work to create a new IPP category so eventually, this will move there.